project/plugins.sbt

```diff
@@ -21,3 +21,5 @@ addSbtPlugin("com.typesafe" % "sbt-mima-plugin" % "1.1.2")
 addSbtPlugin("ch.epfl.scala" % "sbt-tasty-mima" % "1.0.0")
 
 addSbtPlugin("com.github.sbt" % "sbt-native-packager" % "1.10.0")
+
+addSbtPlugin("com.gradle" % "sbt-develocity" % "1.0.1")
```

Comment:
Isn't `1.1` the latest version?

Reply:
Correct. But it wasn't available when this PR was initially opened.  We'll have v1.1.1 published next week, as it's needed to address the problem Adrien faced with the output redirection.